test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -41,7 +41,7 @@ public OnnxConversionTest(ITestOutputHelper output) : base(output)
         /// call <see cref="OnnxScoringEstimator"/> to evaluate that file. The outputs of <see cref="OnnxScoringEstimator"/> are checked against the original
         /// ML.NET model's outputs.
         /// </summary>
-        [Fact]
+        [ConditionalFact(typeof(BaseTestBaseline), nameof(BaseTestBaseline.NotFullFramework))]
```

Comment:
Can you add a link to [https://github.com/dotnet/machinelearning/issues/2106](https://github.com/dotnet/machinelearning/issues/2106) here?